bundles/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/discovery/SoulissGatewayDiscovery.java

```diff
@@ -0,0 +1,257 @@
+                    label = "T55: node " + node + ", slot " + slot;
+                    break;
+                case SoulissProtocolConstants.SOULISS_T56_CURRENT_SENSOR:
+                    thingUID = new ThingUID(SoulissBindingConstants.T56_THING_TYPE, gatewayUID, sNodeId);
+                    label = "T56: node " + node + ", slot " + slot;
+                    break;
+                case SoulissProtocolConstants.SOULISS_T57_POWER_SENSOR:
+                    thingUID = new ThingUID(SoulissBindingConstants.T57_THING_TYPE, gatewayUID, sNodeId);
+                    label = "T57: node " + node + ", slot " + slot;
+                    break;
+                case SoulissProtocolConstants.SOULISS_T61:
+                    thingUID = new ThingUID(SoulissBindingConstants.T61_THING_TYPE, gatewayUID, sNodeId);
+                    label = "T61: node " + node + ", slot " + slot;
+                    break;
+                case SoulissProtocolConstants.SOULISS_T62_TEMPERATURE_SENSOR:
+                    thingUID = new ThingUID(SoulissBindingConstants.T62_THING_TYPE, gatewayUID, sNodeId);
+                    label = "T62: node " + node + ", slot " + slot;
+                    break;
+                case SoulissProtocolConstants.SOULISS_T63_HUMIDITY_SENSOR:
+                    thingUID = new ThingUID(SoulissBindingConstants.T63_THING_TYPE, gatewayUID, sNodeId);
+                    label = "T63: node " + node + ", slot " + slot;
+                    break;
+                case SoulissProtocolConstants.SOULISS_T64_LUX_SENSOR:
+                    thingUID = new ThingUID(SoulissBindingConstants.T64_THING_TYPE, gatewayUID, sNodeId);
+                    label = "T64: node " + node + ", slot " + slot;
+                    break;
+                case SoulissProtocolConstants.SOULISS_T65_VOLTAGE_SENSOR:
+                    thingUID = new ThingUID(SoulissBindingConstants.T65_THING_TYPE, gatewayUID, sNodeId);
+                    label = "T65: node " + node + ", slot " + slot;
+                    break;
+                case SoulissProtocolConstants.SOULISS_T66_CURRENT_SENSOR:
+                    thingUID = new ThingUID(SoulissBindingConstants.T66_THING_TYPE, gatewayUID, sNodeId);
+                    label = "T66: node " + node + ", slot " + slot;
+                    break;
+                case SoulissProtocolConstants.SOULISS_T67_POWER_SENSOR:
+                    thingUID = new ThingUID(SoulissBindingConstants.T67_THING_TYPE, gatewayUID, sNodeId);
+                    label = "T67: node " + node + ", slot " + slot;
+                    break;
+            }
+            if (thingUID != null) {
+                label = "[" + gwHandler.getThing().getUID().getAsString() + "] " + label;
+
+                discoveryResult = DiscoveryResultBuilder.create(thingUID).withLabel(label).withProperty("node", node)
+                        .withProperty("slot", slot).withBridge(gwHandler.getThing().getUID()).build();
```

Comment:
don't think so... the unique combo could be only node + slot. What to do ?

Reply:
You could make it a single property for node and slot, like you already did for the Thing ID. Otherwise, all devices are added to the inbox again, when a discovery is run. Of course it seems more intuitive to have separate config parameters for node and slot. I leave it up to you which solution to choose. If anything is still unclear, don't hesitate to ask.

Reply:
added uniqueId (ip of gateway+Node+Slot) as representationProperty 
https://github.com/openhab/openhab-addons/pull/11083/commits/ea421a0364b9d3b74576e8b1d0ee84451ad0ec69

Reply:
Is the unique ID updated when the user updates the configuration with another node/slot?

The representation property needs to be unique only bridge wide. So, the gateway could be removed. See https://next.openhab.org/docs/developer/bindings/#representation-property